src/main/scripts/add_bam_pair_xenocp.py

```diff
@@ -118,12 +118,11 @@ def add_bam_pair_stp(raptr, sample, target, project, subproject, **kwargs):
     raptr.execute(query, (bam_id_xenocp, bam_tpl_id_xenocp))
 
     # Get read_group_ids and insert into bam_read_group
-    query = """SELECT read_group_id, pu from read_group inner join (select loadable_id as lid, 
-            project_id as pid, project_name, subproject, target_name from loadable inner join
-            (select * from sample_target_project_view where
-            formal_name = %s and target_name = %s and project_name = %s 
-            and subproject = %s) using (sample_target_id))
-            on loadable_id = lid and project_id = pid where status = 'Normal';"""
+    query = """select rg.read_group_id, rg.pu
+            from read_group rg inner join sample_target_project_view stpv
+            on rg.sample_target_id = stpv.sample_target_id and rg.project_id = stpv.project_id
+            where status = 'Normal' and formal_name = %s and target_name = %s
+            and project_name = %s and subproject = %s;"""
```

Comment:
Let's take the opportunity to format this (i.e. newlines, indentation, etc.).  It's already changed enough that the whole block is a diff, so no reason not to fix the lack of formatting of the query.

Reply:
Something like this?

```python
    query = """select
            rg.read_group_id,
            rg.pu
        from
            read_group rg
            inner join sample_target_project_view stpv
                on rg.sample_target_id = stpv.sample_target_id and rg.project_id = stpv.project_id
        where
            status = 'Normal'
            and formal_name = %s
            and target_name = %s
            and project_name = %s
            and subproject = %s;"""
```